resources/js/features/forums/components/ShowForumTopicMainRoot/ShowForumTopicMainRoot.tsx

```diff
@@ -83,11 +89,20 @@ export const ShowForumTopicMainRoot: FC = () => {
       <div>
         {auth?.user.isMuted && auth.user.mutedUntil ? (
           <MutedMessage mutedUntil={auth.user.mutedUntil} />
-        ) : (
+        ) : null}
+
+        {forumTopic.lockedAt ? (
+          <div className="flex select-none items-center justify-center gap-1.5 bg-embed p-2 text-center text-neutral-300 transition light:text-neutral-900">
+            <LuLock className="size-4" />
+            <p>{t('This topic is locked.')}</p>
+          </div>
+        ) : null}
```

Comment:
Minor: if not logged in, it still says you can join the conversation by logging in.
![image](https://github.com/user-attachments/assets/ff4a3a40-2d03-4ee0-becc-2ad275db39be)

Reply:
Fixed in latest.